joomla/joomla-cms
# Templates search tools ordering

#### Description

This PR adds, for consistency, as it exists in so many other views in the backoffice, the order select box of the search tools to com_templates.
###### After PR

![image](https://cloud.githubusercontent.com/assets/9630530/12926817/fa3ce3cc-cf5c-11e5-8df3-442e526342bd.png)

![image](https://cloud.githubusercontent.com/assets/9630530/12926841/145d18ee-cf5d-11e5-8586-212eec647984.png)
#### How to test
1. Install latest staging and apply this patch
2. Go to Extensions -> Templates
3. Check the ordering select box is working fine in "Styles" and in "Templates" view
